src/platform/device.gni

```diff
@@ -23,6 +23,9 @@ declare_args() {
 
   # Substitute fake platform when building with chip_device_platform=auto.
   chip_fake_platform = false
+
+  # Use actual device attestation credentials
+  chip_device_attestation_credentials = false
```

Comment:
Please use `chip_device_use_example_attestation_credentials = true` and make also a define for it. it will be clearer